third_party/move/move-vm/runtime/src/loader/mod.rs

```diff
@@ -397,6 +382,7 @@ impl LoaderV1 {
     // Entry point for script execution (`MoveVM::execute_script`).
     // Verifies the script if it is not in the cache of scripts loaded.
     // Type parameters are checked as well after every type is loaded.
+    #[allow(dead_code)]
```

Comment:
can't we delete?

Reply:
Will delete in later PR (don't remember which one😅), where basically the whole V1 gets deleted (so keeping here it's ok)